### PR TITLE
Feature/create colors

### PR DESCRIPTION
**Pull request by kamskr**
## Description

1. Create lou_bank_ui with colors
2. Create lou_bank_gallery

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: packages/lou_bank_ui/lib/src/typography/lou_typography.dart [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+abstract class LouTypography {
+  static const _baseTextStyle = TextStyle(
+      fontWeight: LouFontWeight.medium,
+      height: 1.25,
+      package: 'lou_bank_ui',
+      fontFamily: 'Roboto');
+
+  static final TextStyle headline1 =
+      _baseTextStyle.copyWith(fontSize: 25, fontWeight: LouFontWeight.bold);
```
Comment (jdebecka) [inline packages/lou_bank_ui/lib/src/typography/lou_typography.dart]: comma after bold, formatting is odd

File: packages/lou_bank_ui/lib/src/typography/lou_typography.dart [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+  static final TextStyle headline2 =
```
Comment (jdebecka) [inline packages/lou_bank_ui/lib/src/typography/lou_typography.dart]: comma, and it is a repetition of h1

File: packages/lou_bank_ui/lib/src/typography/lou_typography.dart [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      _baseTextStyle.copyWith(fontSize: 25, fontWeight: LouFontWeight.bold);
+}
+
+abstract class LouFontWeight {
+  static const FontWeight medium = FontWeight.w500;
+  static const FontWeight bold = FontWeight.bold;
```
Comment (jdebecka) [inline packages/lou_bank_ui/lib/src/typography/lou_typography.dart]: w700/w900 inconsistent

File: packages/lou_bank_ui/lou_bank_gallery/lib/list_screen.dart [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: ListView(
+          children: _tileTitles.entries
+              .map(
+                (entry) => ListTile(
+                  onTap: () => Navigator.of(context).push(
+                      MaterialPageRoute(builder: (context) => entry.value)),
```
Comment (jdebecka) [inline packages/lou_bank_ui/lou_bank_gallery/lib/list_screen.dart]:
```suggestion
                      MaterialPageRoute(builder: (context) => entry.value,),),
```

File: packages/lou_bank_ui/lou_bank_gallery/lib/main.dart [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: Scaffold(
+        appBar: AppBar(title: const Text('Gallery')),
```
Comment (jdebecka) [inline packages/lou_bank_ui/lou_bank_gallery/lib/main.dart]:
```suggestion
        appBar: AppBar(title: const Text('Gallery',),),
```

File: packages/lou_bank_ui/lou_bank_gallery/lib/theme/color_screen.dart [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('Colors')),
+      body: ListView(children: <Widget>[
+        ListTile(
+          title: const Text('white = Colors.white'),
+          trailing: SizedBox(
+            width: 42.0,
+            height: 42.0,
```
Comment (jdebecka) [inline packages/lou_bank_ui/lou_bank_gallery/lib/theme/color_screen.dart]: repetition. Take it out to const var

File: packages/lou_bank_ui/lou_bank_gallery/lib/theme/color_screen.dart [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+                  color: LouColors.white,
+                  border: Border.all(color: Colors.black)),
+            ),
+          ),
+        ),
+        ListTile(
+          title: const Text('yellow = Color(0xffF2FE8D)'),
+          trailing: SizedBox(
+            width: 42.0,
+            height: 42.0,
```
Comment (jdebecka) [inline packages/lou_bank_ui/lou_bank_gallery/lib/theme/color_screen.dart]: Maytbe do file with dimentions. It seems like you use 42 a lot

File: packages/lou_bank_ui/lou_bank_gallery/lib/theme/color_screen.dart [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+            child: DecoratedBox(
+              decoration: BoxDecoration(
+                  color: LouColors.yellow,
+                  border: Border.all(color: Colors.black)),
+            ),
+          ),
+        ),
+        ListTile(
```
Comment (jdebecka) [inline packages/lou_bank_ui/lou_bank_gallery/lib/theme/color_screen.dart]: List tile is being repeated. MAke a widget out of it that will take things that change as a parameter.

File: packages/lou_bank_ui/lou_bank_gallery/lib/theme/color_screen.dart [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+          trailing: SizedBox(
+            width: 42.0,
+            height: 42.0,
+            child: DecoratedBox(
+              decoration: BoxDecoration(
+                  border: Border.all(color: Colors.black),
+                  gradient: const LinearGradient(
+                      begin: Alignment.topCenter,
+                      end: Alignment.bottomCenter,
+                      colors: [LouColors.grey1Start, LouColors.grey1End])),
```
Comment (jdebecka) [inline packages/lou_bank_ui/lou_bank_gallery/lib/theme/color_screen.dart]: missing commans. Code will not formayt without them

File: packages/lou_bank_ui/lou_bank_gallery/lib/theme/color_screen.dart [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+            child: DecoratedBox(
+              decoration: BoxDecoration(
+                  border: Border.all(color: Colors.black),
+                  gradient: const LinearGradient(
+                      begin: Alignment.topCenter,
+                      end: Alignment.bottomCenter,
+                      colors: [
+                        LouColors.gradientLilacStart,
+                        LouColors.gradientLilacEnd
+                      ])),
```
Comment (jdebecka) [inline packages/lou_bank_ui/lou_bank_gallery/lib/theme/color_screen.dart]: commas

File: packages/lou_bank_ui/lou_bank_gallery/test/widget_test.dart [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+//
+// To perform an interaction with a widget in your test, use the WidgetTester
+// utility that Flutter provides. For example, you can send tap and scroll
+// gestures. You can also use WidgetTester to find child widgets in the widget
+// tree, read text, and verify that the values of widget properties are correct.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {}
```
Comment (jdebecka) [inline packages/lou_bank_ui/lou_bank_gallery/test/widget_test.dart:11]: missing testes :)

File: packages/lou_bank_ui/pubspec.yaml [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+  flutter_test:
+    sdk: flutter
+
+include: package:very_good_analysis/analysis_options.yaml
+
+flutter:
+  fonts:
+    - family: Roboto
+      fonts:
+        - asset: assets/fonts/Roboto-Regular.ttf
```
Comment (jdebecka) [inline packages/lou_bank_ui/pubspec.yaml:27]: define weight

File: packages/lou_bank_ui/lou_bank_gallery/test/widget_test.dart (hunk 11 above)
Comment (kamskr) [reply packages/lou_bank_ui/lou_bank_gallery/test/widget_test.dart:11]: I will add them after creating the theme

**Review by jdebecka — APPROVED** [review]

**Pull request merged**